src/Tickets/Commerce/Gateways/Stripe/Client.php

```diff
@@ -598,7 +711,7 @@ public function calculate_application_fee_value( Value $value ) {
 	 */
 	public function get_payment_method_types() {
 
-		if ( Settings::CARD_ELEMENT_SLUG === tribe_get_option( Settings::$option_checkout_element ) ) {
+		if ( $this->payment_element_fallback_mode || Settings::CARD_ELEMENT_SLUG === tribe_get_option( Settings::$option_checkout_element ) ) {
```

Comment:
Create a method that exposes the fallback mode externally. `$this->is_payment_element_fallback_mode()`